include/messages.h

```diff
@@ -234,6 +240,8 @@ enum struct ContentType : uint8_t
 struct Proposal
 {
   std::variant<Add, Update, Remove> content;
+
+  static const ContentType type;
   TLS_SERIALIZABLE(content)
   TLS_TRAITS(tls::variant<ProposalType>)
```

Comment:
where are we serializing ContentType?

Reply:
That gets handled by the `tls::variant<ContentType>` inside `struct MLSPlaintext`